screens/auth/LogInScreen.js

```diff
@@ -48,6 +50,7 @@ export default class LogInScreen extends React.Component {
   // to be the user ID and then navigates to homescreen.
   _asyncLogIn = async userId => {
     await AsyncStorage.setItem('userId', userId);
+    Sentry.captureMessage('Log In Successful');
```

Comment:
If it's possible to log `userId` in this message, it'll be a much more valuable message; also I might guess that `Log In Unsuccessful` for a particular case would be rarer (hopefully) and something we are actually looking for

Reply:
It still does capture the user info in line 95 so these messages are also attached to the user details.

Reply:
@wangannie @JustinMi -- with Firebase analytics tracking these changes, i would also suggest removing this and having Sentry focus more on capturing error states